server/test/buff_server/packages_test.exs

```diff
@@ -28,63 +42,79 @@ defmodule BuffServer.PackagesTest do
       repository_url: nil
     }
 
-    def package_fixture(attrs \\ %{}) do
-      {:ok, package} =
-        attrs
-        |> Enum.into(@valid_attrs)
-        |> Packages.create_package()
+    def package_fixture(attrs) do
+      ExAws.Request.HttpMock
+      |> expect(:request, fn _method, _url, _body, _headers, _opts -> {:ok, %{status_code: 200}} end)
 
-      package
+      attrs
```

Comment:
yeah but attrs aren't the default_attrs.. they're what I merge with them, it looks confusing because I define default_attrs there also, I'll just move that to a function